### PR TITLE
Add "All" option so that the staffer summary

### LINKED ISSUE #1392: Staffer Summary
This is related to #1375 / #1385 
I love the new layout to be able to see all pertinent data for staffers.
Currently is it broken down by department:
https://classic.uber.magfest.org/uber/jobs/staffers

Can we create a page, (or am I missing it somewhere) that would show all this data for all staffers.
Once Sorting is figured out, that will allow us to quickly sort and Identify any staffers that haven't completed the info.

### PR DESCRIPTION
**Pull request by Lusarien**
can be viewed across all departments fixed #1392

### CODE REVIEW

File: uber/site_sections/jobs.py [hunk 1]
```diff
@@ -172,6 +177,15 @@ def staffers_by_job(self, session, id, message=''):
                                 .order_by(Attendee.full_name).all()
         }
 
+    def allStaffers(self, session, id, message=''):
+        job = session.job(id);
```
Comment (EliAndrewC) [inline uber/site_sections/jobs.py]: ``pep8`` is complaining that this line ends with a semicolon, which is why Travis says that the build failed.  Can you remove that and push out the new commit to this branch?  That will automatically update the Pull Request.

File: uber/templates/jobs/staffers.html [hunk 2]
```diff
@@ -49,6 +49,7 @@
     <th> Hours Total </th>
     <th> Checked In</th>
 </tr>
+</thead>
```
Comment (EliAndrewC) [inline uber/templates/jobs/staffers.html]: This was an unrelated fix.  @Lusarien noticed that ``datatables`` was putting icons in the wrong place, and then figured out that it was because we weren't closing the ``<thead>``.

File: uber/site_sections/jobs.py (hunk 1 above)
Comment (Lusarien) [inline uber/site_sections/jobs.py]: That function actually isn't even supposed to exist... was a dummy function for troubleshooting, just removing the whole thing good catch!

**Comment by EliAndrewC** [comment]
Good stuff!
